Sources/StreamChat/Workers/ReactionListUpdater.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+class ReactionListUpdater: Worker {
```

Comment:
Thinking if we should keep it next to the old load reactions method in `MessageUpdater`.